Library/Formula/cjdns.rb

```diff
@@ -0,0 +1,13 @@
+class Cjdns < Formula
+  desc "Advanced mesh routing with cryptographic addressing and integrity/confidentiality protection"
+  homepage "https://github.com/cjdelisle/cjdns/"
+  url "https://github.com/cjdelisle/cjdns/archive/cjdns-v16.1.tar.gz"
+  sha256 "3eebb276ee63f103c54aa9201e67d8d810ae3a88af90300dd3a3a894e73cb400"
+
+  depends_on "node" => :build
+
+  def install
+    system './do'
+    bin.install 'cjdroute'
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!